### ISSUE TITLE
Less file with .background-image doesn't comile.

### ISSUE THREAD

**Issue by rantiev** [madebysource/lesshat#98]
Hi, guys, i'm trying to compile file that has mixin with JS evaluation, but it doesn't compile. How could you get it work, i can't see... I was trying to compile it using Webstorm "less css compiler" plugin then i started to try comile using node.js module "less" and "simple less" windows client side comiler. Nothing helps.

If i try to use simple .transition mixin, it works. 

my file contents only import and 1 .background-image mixin call (from your docs).

@import "lesshat";
div {
    .background-image(linear-gradient(to bottom, #fb83fa 0%,#e93cec 100%));
}

That's all.

**Comment by BarbzYHOOL** [comment]
Still an issue 4 years later on lesshat 3.x

any solution?


Edit: ok i can update to lesshat4 but it seems VERY different and harder to install

**Comment by mboudreau** [comment]
Yeah, switch to sass. Less development pretty much stopped, sass is better.

**Comment by BarbzYHOOL** [comment]
same amount of commits and contributors in the past few days on less / sass
